docs/DeveloperGuide.md

```diff
@@ -334,27 +334,50 @@ This is to prevent `SortCommand` from taking on more responsibilities (Separatio
     * Pros: Easy to implement, controlled and less likely to be used incorrectly. This increase ease of use for users.
     * Cons: Limited sorting and lesser functionality.
 
-### Meeting feature
+
+### Meeting
 
 Meeting is feature that allows the user to keep track of any events they may have with the particular contact. It contains the description of the meeting event with the date and time it would occur.
 
-#### How the feature is implemented
+#### Implementation
```

Comment:
not sure if this is a mistake, but this subheading was supposed to be updated to the standardized "How the feature is implemented" right?